apps/atomy-q/API/tests/Unit/Database/Factories/UserFactoryTest.php

```
@@ -0,0 +1,33 @@
+
+use App\Models\User;
+use Tests\TestCase;
+
+final class UserFactoryTest extends TestCase
+{
+    public function test_user_active_state(): void
+    {
+        $user = User::factory()->active()->create();
+
+        $this->assertSame('active', $user->status);
+    }
```

PHPMD (2.15.0)

[error] The method test_user_active_state is not named in camelCase. (undefined)

(CamelCaseMethodName)